index.js

```diff
@@ -54,19 +54,24 @@ module.exports = {
       let checker = new VersionChecker(this.parent).for('ember-cli-babel', 'npm');
 
       if (checker.satisfies('>= 6.0.0')) {
+        let addons = this._getConfig().coveredAddons.map((addonName) => this.project.findAddonByName(addonName));
+        const addonIncludes = addons
+              .map((addon) => this._getIncludesForDir(path.join(addon.root, 'addon'), addon.name))
+              .filter(Boolean)
+              .reduce((acc, val) => acc.concat(val));
```

Comment:
External addons don't get added to coverage, but because all our code lives in them, we want certain ones.  Therefore I look up the addons here and add their source files in `addon` as a big array to the instanbul includes.